std_msgs/Header header
ObjectSegment[] objects
